### PR TITLE
Fix quiz 13 and add Playwright test

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#25 by:
1. Adding subtitles and correct ID's to each exercise.
2. Translating the whole file to Spanish.
3. Adding a `playwright` test and formatting with `black`.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

https://user-images.githubusercontent.com/80920858/121946347-745c9380-cd1a-11eb-95c2-f41df52a5072.mp4

### CODE REVIEW

File: _sources/quiz/Quiz13.rst [hunk 1]
```diff
@@ -7,91 +7,118 @@ Quiz - 13
...
+Ejercicio 1
+-----------
 
-   A. fim_igual |br|
-   Dada uma lista de strings, retorna o número de strings |br|
-   com tamanho >= 2 onde o primeiro e o último caracteres são iguais |br|
+.. activecode:: q13_1
+   :nocodelens:
 
+   Desarrolle la función ``terminan_igual`` que, dada una lista de cadenas, devuelva el número de cadenas
```
Comment (y-martinez) [inline _sources/quiz/Quiz13.rst:16]:
```suggestion
   Desarrolle la función ``terminan_igual`` que, dada una lista de cadenas ``palabras``, devuelva el número de cadenas
```

File: _sources/quiz/Quiz13.rst [hunk 2]
```diff
@@ -7,91 +7,118 @@ Quiz - 13
...
    :nocodelens:
 
-   B. x_antes |br|
-   Dada uma lista de strings retorna uma lista onde todos os elementos |br|
-   que começam com x ficam sorteados antes  |br|
-   Exemplo ['mix', 'xyz', 'apple', 'xanadu', 'aardvark'] retorna |br|
-   ['xanadu', 'xyz', 'aardvark', 'apple', 'mix'] |br|
-   Dica: monte duas listas separadas e junte-as no final |br|
+   Desarrolle la función ``x_antes`` que, dada una lista de cadenas ``palabras``, devuelva 
+   otra lista en la que todos los elementos que comienzan con ``"x"`` sean los primeros en la lista.
```
Comment (y-martinez) [inline _sources/quiz/Quiz13.rst:47]:
```suggestion
   otra lista en la que todas las cadenas que comienzan con el carácter ``'x'`` sean los primeros elementos de la lista. 
```

**Review by y-martinez — CHANGES_REQUESTED** [review]
I left some little suggestions 👍🏼

**Review by NicolasSandoval — APPROVED** [review]
Great work!

**Pull request merged**